gobblin-runtime/src/main/java/gobblin/scheduler/JobScheduler.java

```diff
@@ -116,45 +116,57 @@
 
   private final boolean waitForJobCompletion;
 
-  public JobScheduler(Properties properties) throws Exception {
+  public JobScheduler(Properties properties)
+      throws Exception {
     this.properties = properties;
     this.scheduler = new StdSchedulerFactory().getScheduler();
 
-    this.jobExecutor = Executors.newFixedThreadPool(
-        Integer.parseInt(properties.getProperty(ConfigurationKeys.JOB_EXECUTOR_THREAD_POOL_SIZE_KEY,
+    this.jobExecutor = Executors.newFixedThreadPool(Integer.parseInt(
+        properties.getProperty(ConfigurationKeys.JOB_EXECUTOR_THREAD_POOL_SIZE_KEY,
             Integer.toString(ConfigurationKeys.DEFAULT_JOB_EXECUTOR_THREAD_POOL_SIZE))),
         ExecutorsUtils.newThreadFactory(Optional.of(LOG), Optional.of("JobScheduler-%d")));
 
-    this.jobConfigFileExtensions =
-        Sets.newHashSet(Splitter.on(",").omitEmptyStrings().split(this.properties.getProperty(
-            ConfigurationKeys.JOB_CONFIG_FILE_EXTENSIONS_KEY, ConfigurationKeys.DEFAULT_JOB_CONFIG_FILE_EXTENSIONS)));
+    this.jobConfigFileExtensions = Sets.newHashSet(Splitter.on(",")
+        .omitEmptyStrings()
+        .split(this.properties.getProperty(ConfigurationKeys.JOB_CONFIG_FILE_EXTENSIONS_KEY,
+            ConfigurationKeys.DEFAULT_JOB_CONFIG_FILE_EXTENSIONS)));
 
-    long pollingInterval =
-        Long.parseLong(this.properties.getProperty(ConfigurationKeys.JOB_CONFIG_FILE_MONITOR_POLLING_INTERVAL_KEY,
+    long pollingInterval = Long.parseLong(
+        this.properties.getProperty(ConfigurationKeys.JOB_CONFIG_FILE_MONITOR_POLLING_INTERVAL_KEY,
             Long.toString(ConfigurationKeys.DEFAULT_JOB_CONFIG_FILE_MONITOR_POLLING_INTERVAL)));
     this.fileAlterationMonitor = new FileAlterationMonitor(pollingInterval);
 
-    this.waitForJobCompletion =
-        Boolean.parseBoolean(this.properties.getProperty(ConfigurationKeys.SCHEDULER_WAIT_FOR_JOB_COMPLETION_KEY,
+    this.waitForJobCompletion = Boolean.parseBoolean(
+        this.properties.getProperty(ConfigurationKeys.SCHEDULER_WAIT_FOR_JOB_COMPLETION_KEY,
             ConfigurationKeys.DEFAULT_SCHEDULER_WAIT_FOR_JOB_COMPLETION));
   }
 
   @Override
-  protected void startUp() throws Exception {
+  protected void startUp()
+      throws Exception {
     LOG.info("Starting the job scheduler");
-
     this.scheduler.start();
-    if (this.properties.containsKey(ConfigurationKeys.JOB_CONFIG_FILE_DIR_KEY)) {
-      scheduleLocallyConfiguredJobs();
-      startJobConfigFileMonitor();
+
+    Preconditions.checkArgument(
```

Comment:
@abti The monitor part will break the gobblin-cluster for now but that should affect `monitor-dev` branch. Will talk offline.